druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/RocksDBManager.java

```diff
@@ -129,8 +129,8 @@ public String createDB(final RocksDBExtractionNamespace extractionNamespace,
             FileUtils.forceMkdir(file);
         }
 
-        final String localZippedFileNameWithPath = String.format("%s/%s/%s/rocksdb_%s.zip",
-                localStorageDirectory, extractionNamespace.getNamespace(), loadTime, UUID.randomUUID().toString());
+        final String localZippedFileNameWithPath = String.format("%s/%s/%srocksdb_%s.zip",
+                localStorageDirectory, extractionNamespace.getNamespace(), loadTime, getLocalPathSuffix(extractionNamespace.isRandomLocalPathSuffixEnabled()));
```

Comment:
Did you notice that there is one less slash in change, zip file is stored in parent dir with prefix as loadTime? Looks like bug. Need to add slash after load time

Reply:
I removed the slash intentionally as the path suffix could be empty. The specific logic is on this line https://github.com/yahoo/maha/blob/0d76a8b08112e26dfbd399afb2b3942761c33a7c/druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/RocksDBManager.java#L165